src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline - a task with a due date.
+ */
+public class Deadline extends Task {
+    private LocalDate by;
+
+    /**
+     * Creates a new Deadline.
```

Comment:
Very minor thing, but following CS2103T's Java coding standard, 
> Empty line between description and parameter section 

they would require an empty line here. However, if the checkstyle script (in week 4's increment A-checkstyle) did not raise this up, I believe it would be fine to leave it as it is.